docker-compose-quickstart.yml

```diff
@@ -0,0 +1,31 @@
+# Inspired by https://docs.docker.com/compose/django/
+version: '3'
+
+services:
+  db:
+    # NOTE: No persistance storage configured.
+    # See: https://hub.docker.com/_/postgres/
+    image: postgres
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust
+    volumes:
+      - ./docker-init-db.sql:/docker-entrypoint-initdb.d/init_db.sql
+
+  cache:
+    image: redis
+
+  web:
+    image: maykinmedia/record-management-app:latest
+    environment:
+      - DJANGO_SETTINGS_MODULE=rma.conf.docker
+      - ALLOWED_HOSTS=*
+      - DEBUG=True
+      - CACHE_DEFAULT=cache:6379/0
+      - CACHE_AXES=cache:6379/0
+      - CACHE_OAS=cache:6379/1
+      - SECRET_KEY=${SECRET_KEY:-145cqii@)+z$b&9dcmdctpm+&be1#9zslxm+x)0+u*^qf!qz(6}
+    ports:
+      - 8000:8000
+    depends_on:
+      - db
+      - cache
```

Comment:
Sorry, but one more thing, which I've just noticed - all docker-compose files lacks celery containers, which are needed to perform the selected zaken removal.
You can look at BPTL docker-compose file for some inspiration.